mungegithub/mungers/lgtm_handler.go

```diff
@@ -29,44 +33,56 @@ const (
 	lgtmRemovedBody = "PR changed after LGTM, removing LGTM."
 )
 
-// LGTMAfterCommitMunger will remove the LGTM flag from an PR which has been
-// updated since the reviewer added LGTM
-type LGTMAfterCommitMunger struct{}
+// LGTMHandler will
+// - apply the LGTM label if reviewer has said so, or
+// - remove the LGTM label from an PR which has been updated since the reviewer added LGTM
```

Comment:
Reviewers should also be able to remove lgtm's through comments since they will not have write access, and may accidentally add one or notice something later.